lib/cli/cli.js

```diff
@@ -98,15 +98,17 @@ CLI.prototype.run = function(environment) {
 
     var PlatformChecker = require('../utilities/platform-checker');
     var platform = new PlatformChecker(process.version);
-    if (!platform.isValid && !this.testing) {
+    var recommendation = ' We recommend that you use the most-recent "Active LTS" version of Node.js.';
+
+    if (!this.testing) {
       if (platform.isDeprecated) {
         this.ui.writeDeprecateLine('Node ' + process.version +
-                                   ' is no longer supported by Ember CLI. Please update to a more recent version of Node');
+                                   ' is no longer supported by Ember CLI.' + recommendation);
       }
 
-      if (platform.isUntested) {
+      if (!platform.isTested) {
         this.ui.writeWarnLine('Node ' + process.version +
-                              ' has currently not been tested against Ember CLI and may result in unexpected behaviour.');
+                              ' is not tested against Ember CLI on your platform.' + recommendation);
```

Comment:
I'm wondering why we differentiate between tested and deprecated here. IMHO the deprecation message based on the engine compat field in `package.json` should be enough.